### PR TITLE
Add models and get the tests running

### PR DESCRIPTION
**Pull request by adam-thomas**
Copied from an internal project with a lot of renaming.

@incuna/backend review please?

### CODE REVIEW

**Comment by adam-thomas** [comment]
Hooray, it works now!

File: surveys/tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        This will throw an error if answers=[] is not allowed.
+
+        Without accepting answers=[] on UserResponse, we can't have non-required
+        questionnaire form fields.
+        """
+        factories.UserResponseFactory.create(answers=[])
+
+    def test_num_users(self):
+        factories.UserResponseFactory.create_batch(2, user_id='first_user')
+        factories.UserResponseFactory.create(user_id='second_user')
```
Comment (KevinEtienne) [inline surveys/tests/test_models.py]: I would import factories (`SurveyFieldFactory`, `UserResponseFactory` and `SurveyFieldsetFactory`) and models  (`SurveyField`, `UserResponse`, `SurveyFieldset`) directly to make expressions shorter where we use them.

**Comment by KevinEtienne** [comment]
Looks good.

File: surveys/tests/test_models.py (hunk 1 above)
Comment (adam-thomas) [inline surveys/tests/test_models.py]: Ah I hate editing imports like that, and having to manually keep everything in alphabetical order, and swap them between single- and multi-line...

**Comment by adam-thomas** [comment]
Updated.

**Comment by adam-thomas** [comment]
Hmm actually

**Comment by adam-thomas** [comment]
Comparing the code at https://github.com/incuna/incuna-surveys/pull/1/commits/859cbae14e2b167c10502942346138872ddc30c2 and https://github.com/incuna/incuna-surveys/pull/1/commits/dc12eefe9a0a5ed8d8ee044db413c4e3c575f429, I feel like the latter is simpler, cleaner and easier to update.  When you want to add a new test class you don't have to fiddle around with the imports (and ensure they stay within 90 characters).  You also get to benefit from autocorrect popping up when you type `models.` or `factories.` :)

**Comment by adam-thomas** [comment]
New day bump :)

**Comment by KevinEtienne** [comment]
Looks good to me, and I feel it's cool to have your point on import, let me give my feeling about targeted import briefly:
 - it doesn't add more than what we need (overriding things in the current scope) - however our scope are relatively small;
 - might avoid importing extra things for other modules (module A import everything from module B which import everything from module C)
 - less verbose (I use the unix tool sort to sort multi-lines import) - however I see that verbose import and whole module import in your last commit contains the same amount of chars.

**Comment by KevinEtienne** [comment]
Ooh I can't merge :)

**Pull request merged**